packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -57,6 +57,8 @@ export const messages = {
     'Enter an org alias or use default alias',
   parameter_gatherer_enter_custom_url:
     'Enter a custom login URL or use default URL',
+  parameter_gatherer_enter_scratch_org_def_files:
```

Comment:
@hbc-bindu Please let me know if you'd like to update these messages to the user.

Reply:
Hi @sfsholden, tiny change:
'Enter an org alias or use the default alias'
'Enter a custom login URL or use the default URL'

Reply:
This piece is specifically for grabbing the scratch def file. The prompts for an org alias and custom login URL are later in the workflow

Reply:
The last gif in the description has an example of that full workflow :)

Reply:
@hbc-bindu Hey sorry I misunderstood - I will add your suggestions to those old messages. Does everything look good for the new messages?